packages/react-scripts/scripts/utils/frontierInit.js

```diff
@@ -197,10 +197,21 @@ function buildInstallCommandAndArgs(useYarn, saveDev = false) {
   return { command, args };
 }
 
-function sortScripts(scripts){
+function setupEnvars(appPath) {
+  osUtils.runExternalCommandSync('npx', ['@fs/fr-cli', 'env', 'local']);
+  const envPath = path.join(appPath, '.env');
+  let envFile = fs.readFileSync(envPath, 'UTF8');
+
+  envFile += `\nSKIP_PREFLIGHT_CHECK=true`;
+  fs.writeFileSync(envPath, envFile);
```

Comment:
why is it better to modify the file here instead of in the cli command? is this var only needed for the first init, but not at other times when running locally?

Reply:
A .env file exists in the template with just that single variable set to true.

Running `fr env` overrides the existing .env file, so I manually add that back here.

Reply:
Thats a good point. I guess I could put it in the CLI. It doesn't make sense to have it in an environment where it isn't a CRA-made app, but I guess it wouldn't hurt anything in that scenario.